vendor/bat-native-ads/src/bat/ads/internal/features.cc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "bat/ads/internal/features.h"
+
+namespace ads {
+namespace features {
+
+// Controls behaviour of the contextual ad matching mechanism, e.g. by
+// adjusting the number of page classifications used to infer user interest.
+const base::Feature kContextualAdsControl { "ContextualAdsControl",
+    base::FEATURE_DISABLED_BY_DEFAULT };
```

Comment:
clang-format

Reply:
following the convention within bat-native-ads as per @tmancey - iirc this will be changed in all of ads with a future refactor

Reply:
when we have merged some important features we will run clang format across ads. thanks

Reply:
@tmancey for future PRs please follow clang-format conventions. There's no reason to add new code that doesn't follow the standards. Old code can be dealt with later.